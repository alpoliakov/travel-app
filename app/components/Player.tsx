import React from 'react';
import ReactPlayer from 'react-player/youtube';

export default function Player({ videoUrl }) {
  return (
    <div className="player-wrapper w-full bg-gray-900 rounded-xl bg-opacity-50 mx-5 p-5 mb-10 flex justify-center box-border">
      <ReactPlayer
        className="react-player p-5"
        width="100%"
        height="100%"
        url={videoUrl}
        controls
        light
      />
    </div>
  );
}
