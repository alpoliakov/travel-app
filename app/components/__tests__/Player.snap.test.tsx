import React from 'react';
import renderer from 'react-test-renderer';

import Player from '../Player';

it('Player render correctly', () => {
  const tree = renderer.create(<Player videoUrl={'http:'} />).toJSON();

  expect(tree).toMatchSnapshot();
});
