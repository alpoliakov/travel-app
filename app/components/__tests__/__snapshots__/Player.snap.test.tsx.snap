// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player render correctly 1`] = `
<div
  className="player-wrapper w-full bg-gray-900 rounded-xl bg-opacity-50 mx-5 p-5 mb-10 flex justify-center box-border"
>
  <div
    className="react-player p-5"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</div>
`;
